src/snippets/get-started/nuxt/Step3.mdx

```diff
@@ -0,0 +1,43 @@
+import { Code } from "@astrojs/starlight/components";
+import SelectableContent from "@/components/SelectableContent";
+import Step3ConfigJS from "./Step3Config.js?raw";
+import Step3RouteTS from "./Step3Route.ts?raw";
+import Step3RouteJS from "./Step3Route.js?raw";
+import { removeTSCCommentDirectives } from "@/lib/utils";
+
+Update your Nuxt configuration with the contents:
+
+<Code
+  code={removeTSCCommentDirectives(Step3ConfigJS)}
+  lang="js"
+  title="nuxt.config.ts"
+/>
+
+This creates an Arcjet instance that can be used to protect your routes. It can
+be imported from the `arcjet:client` module in your project.
+
+<SelectableContent client:load syncKey="language" frameworkSwitcher>
+<div slot="TS" slotIdx="1">
+
```

Comment:
We'll have to add a 2nd step here where we create `server/utils/arcjet.ts` like this: https://arcjet-docs-git-quinn-nuxt-reference-arcjet.vercel.app/reference/nuxt#configuration